src/backend/TdConsumer.ts

```diff
@@ -34,7 +34,7 @@ export default class TdConsumer {
         this.servient = new Servient();
         this.helper = new Helpers(this.servient);
         // Add all ClientFactories
-        this.servient.addClientFactory(new CoapClientFactory());
+        this.servient.addClientFactory(new CoapClientFactory(new CoapServer()));
```

Comment:
Is this normal? https://github.com/eclipse/thingweb.node-wot/tree/master/packages/binding-coap#client-example does not document this (could be a problem there ofc)

Reply:
Added TODO as we've discussed